CHANGELOG.md

```diff
@@ -26,6 +26,7 @@
 
 ### Mixin
 
+* [CHANGE] Replaced the deprecated `policy/v1beta1` with `policy/v1` when configuring a PodDisruptionBudget for read-write deployment mode. #3811
```

Comment:
Please move below to "Jsonnet" section. I know it's confusing, but "mixin" is alerts + dashboards only.

Reply:
🤦 I meant to put it in the Jsonnet section, not sure why I put it here... thanks for catching this.

Fixed in db12ee2.